x-pack/plugins/ml/server/models/data_recognizer/modules/security_windows/ml/v3_windows_anomalous_script.json

```diff
@@ -0,0 +1,46 @@
+{
+  "job_type": "anomaly_detector",
+  "description": "Looks for unusual powershell scripts that may indicate execution of malware, or persistence mechanisms.",
+  "groups": [
+    "endpoint",
+    "event-log",
+    "process",
+    "windows",
+    "winlogbeat",
+    "powershell"
+  ],
+  "analysis_config": {
+    "bucket_span": "15m",
+    "detectors": [
+      {
+        "detector_description": "high_info_content(\"powershell.file.script_block_text\")",
+        "function": "high_info_content",
+        "field_name": "powershell.file.script_block_text"
+      }
+    ],
+    "influencers": [
+      "host.name",
+      "user.name",
+      "file.Path"
+    ]
+  },
+  "allow_lazy_open": true,
+  "analysis_limits": {
+    "model_memory_limit": "256mb"
+  },
+  "data_description": {
+    "time_field": "@timestamp"
+  },
+  "custom_settings": {
+    "custom_urls": [
```

Comment:
Does it matter that these URLs use `siem#` rather than starting with `security` ?